UploadDaemon/UploadDaemon.cs

```diff
@@ -42,7 +41,7 @@ public static void Main(string[] args)
         {
             if (IsAlreadyRunning())
             {
-                Console.WriteLine("Another instance is already running.");
+                logger.Debug("Another instance is already running. Sending notification to trigger upload.");
```

Comment:
this was written to the console on purpose: it's for the case where the user runs the executable from the command line. please change back to writeline. Logging with debug level will never be seen by anyone ;-)